apps/console/src/features/validation/pages/validation-config-edit.tsx

```diff
@@ -201,6 +215,32 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
             return;
         }
 
+        if (
+            passwordExpiryError.response &&
+            passwordExpiryError.response.data &&
+            passwordExpiryError.response.data.description
+        ) {
+            if (passwordExpiryError.response.status === 404) {
+                return;
+            }
+            dispatch(
+                addAlert({
+                    description:
+                        ValidationConfigStatusFetchRequestError.response.data
+                            .description ??
```

Comment:
If the program reaches this line, that means `ValidationConfigStatusFetchRequestError.response.data.description` is undefined or null (Line 167). Therefore null check is not necessary here.

Reply:
Addressed